src/common/seed/images.ts

```diff
@@ -81,18 +81,31 @@ const generateRandomImage = () => {
 	return selectedVariant;
 };
 
-const generateBuild = () => {
+const remainingStatuses = ['building', 'error', 'ready']; //Is it correct way to make global variable in that function?
+const generateBuild = (statusIndex?: number) => {
 	const statuses = ['building', 'error', 'ready'];
 	const randomNum = Math.round((Math.random() * 2));
 
+	if (statusIndex !== undefined) {
+		const randomStatusIndex = Math.round((Math.random() * (remainingStatuses.length - 1)));
```

Comment:
what the difference between randomStatusIndex and RandomNum? Please, check your code again.

Reply:
- generateDummyImages should only generate random images, no extra logic
- generateDummyImage should receive it's index as an argument from map function in generateDummyImages
- generateDummyImage should pass index in generateBuild function
- depending on generateBuild received index, generate 1 to 3 unique statuses, and rest statuses should be random

this is important, because function is getting twisted and with passing even more arguments it will be hard to manage. We should keep it simple. Please, rewrite logic of the function according to upper notes.